common/src/web_api.rs

```diff
@@ -40,3 +40,12 @@ pub struct UploadKeys {
     pq_last_resort_pre_key: Option<UploadSignedPreKey>,
     signed_pre_key: Option<UploadSignedPreKey>,
 }
+
+#[derive(Serialize, Deserialize)]
+pub struct PreKeyBundle {
```

Comment:
can we please not name structs and other structures the same as things in libsignal?